cmd/conjur.go

```diff
@@ -396,6 +404,7 @@ func init() {
 	conjurLogonCmd.MarkFlagRequired("account")
 	conjurLogonCmd.Flags().StringVarP(&BaseURL, "base-url", "b", "", "Conjur appliance URL")
 	conjurLogonCmd.MarkFlagRequired("base-url")
+	conjurLogonCmd.Flags().StringVarP(&AuthnLDAP, "authn-ldap", "", "", "Uses provided Service ID to configure LDAP Authentication")
```

Comment:
Uses should be User?

Reply:
No, I meant for `Uses` like describing it's action. Do you think it's more coherent to make it `User`?